pages/explore-tracking-plan/start-using-visual-debuggers.mdx

```diff
@@ -31,6 +31,8 @@ Example: `https://example.com/checkout?avo_debug=1`.
 
 ### Accessing mobile debuggers
 
+> If you are using the Inspector SDK on iOS and Android the Visual Debugger is bundled with it, you do not need to do this setup! See `showVisualInspector` and `hideVisualInspector` methods of the Inspector SDKs.
```

Comment:
I feel like this is telling me the setup is optional, and I will only need to do it if I want the visual debuggers? Is that the message we're trying to deliver here? Or is this setup unnecessary if I already have the Inspector SDK initialized? If that's the case, we could maybe frame it like this:

```suggestion
> If you have already initialized the Inspector SDK on iOS and Android, no further action is required to enable the Visual Inspector. See `showVisualInspector` and `hideVisualInspector` methods of the Inspector SDKs to start using the Visual Inspector.
```

Reply:
Makes sense!